xbmc/windowing/X11/GLContextEGL.h

```diff
@@ -12,6 +12,8 @@
 #include "system_egl.h"
 #include "threads/CriticalSection.h"
 
```

Comment:
Need to rebase as this one is in master now.

Reply:
Sorry Heiko, I confused it with [xbmc/windowing/X11/GLContext.h](https://github.com/xbmc/xbmc/commit/6730f62c5d709f8789e11d3f979c597fe702daa3#diff-88ca18ee89c9af8dfb56531ad0018e896873f1a3470778a1a6583945bb837d3b). Apologies!